### DISCUSSION TITLE
Hot Reload C# code

### DISCUSSION THREAD

**Discussion by artkat** [microsoft/WindowsAppSDK#2519] [Q&A]
When will Hot Reload of C# code (not the XAML) be supported?

(or: if it is supported, how to get it working)

Wasting literally hours simply waiting for a rebuild/launch-with-debugger-attached for each and every code edit.

**Comment by kmgallahan** [comment]
Have you read the docs?

https://docs.microsoft.com/visualstudio/debugger/hot-reload?view=vs-2022

**Comment by Marv51** [comment]
In the Packaging projects "Properties" under "Debug", setting the Debugger type to "Managed Only" is really the main point needed. 

For me this is inconvenient, as some Exceptions contain more information if a Hybrid or Native debugger is attached. But I have resigned to switching the debugger type as needed.
